tests/test_dataset_common.py

```diff
@@ -91,7 +92,16 @@ def check_load_dataset(self, dataset_name, configs, is_local=False):
                     is_local=is_local,
                 )
 
-                # build dataset from dummy data
+                if dataset_builder.__class__.__name__ == "Csv":
```

Comment:
@thomwolf - a bit hacky here, but I think it's ok for now

Reply:
Ok! I trust you with the tests :-)